packages/styles/scss/components/callout-quote/_callout-quote.scss

```diff
@@ -20,6 +20,10 @@
 @include quote;
 
 @mixin callout-quote {
+  * {
+    box-sizing: border-box;
+  }
```

Comment:
I can't beleive this got removed from the reset upstream

Reply:
For sure. There are quite a few `box-sizing: border-box` rules building up in the styles package, but ship has sailed to try it overall in a reset, until a major version update probably.